web/main/templates/casebook_page.html

```diff
@@ -23,6 +23,11 @@
 
 <section class="casebook {{casebook_color_class}}">
     <div class="content">
+        <div>
+        {% if section %}
+            {% include 'includes/page_buttons.html'  %}   
+        {% endif %}
+        </div>
```

Comment:
Could this wrapping `<div>` go inside `includes/page_buttons.html` so that there isn't an empty div, when `if section` is false? Or do we need it here all the time?

Sorry to include more questions/etc. outside of the review!

Reply:
No problems thanks for all the comments! added div inside the wrapping!